### PR TITLE
refine rest-server.property documentation

### PR DESCRIPTION
**Pull request by Thespica**
## Purpose of the PR

refine rest-server.property documentation

related to https://github.com/apache/incubator-hugegraph/pull/2610

### CODE REVIEW

File: content/cn/docs/config/config-guide.md [hunk 1]
```diff
@@ -154,6 +154,8 @@ rest-server.properties 文件的默认内容如下：
 
 ```properties
 # bind url
+# http://0.0.0.0:8080 to listen all network interfaces, but need to take care of security
+# or listen to specific IP with port
```
Comment (imbajin) [inline content/cn/docs/config/config-guide.md:158]: same with server

File: content/cn/docs/config/config-guide.md [hunk 2]
```diff
@@ -170,7 +172,7 @@ server.id=server-1
 server.role=master
 ```
 
-- restserver.url：RestServer 提供服务的 url，根据实际环境修改；
+- restserver.url：RestServer 提供服务的 url，根据实际环境修改。如果其他 IP 地址无法访问，可以尝试修改为特定的地址；或修改为 `http://0.0.0.0:8080` 来监听来自任何 IP 地址的请求，这种方案较为便捷，但安全性有所缺失；
```
Comment (imbajin) [inline content/cn/docs/config/config-guide.md:175]: 无需携带端口, 或者 http/https, 只和 ip 有关 

另外"安全性有所缺失" -> "需要留意服务可被访问的网络范围"

File: content/en/docs/config/config-guide.md
Comment (imbajin) [inline content/en/docs/config/config-guide.md:1]: sync with CN version (could also update in hubble's doc?)

Comment (Thespica) [reply content/en/docs/config/config-guide.md:1]: The default bind for [hubble's config](https://github.com/apache/incubator-hugegraph-toolchain/blob/master/hugegraph-hubble/hubble-dist/assembly/static/conf/hugegraph-hubble.properties) is 0.0.0.0, so I think user wouldn't meet this problem about accessing, mean while developers do not need to limit the accessibility of hubble. 

So I think hubble does not need extra explanation.

**Review by imbajin — APPROVED** [review]

**Pull request merged**